src/nft/components/explore/Carousel.tsx

```diff
@@ -65,53 +62,46 @@ interface CarouselProps {
   toggleNextSlide: (idx: number) => void
 }
 
+// todo: do we really need this offset?
 const FIRST_CARD_OFFSET = 0
 
-export const Carousel = ({ children, activeIndex, toggleNextSlide }: CarouselProps) => {
-  const { width } = useWindowSize()
-  const carouselCardContainerRef = useRef<HTMLDivElement>(null)
-  const [cardWidth, setCardWidth] = useState(MAX_CARD_WIDTH)
-
-  useEffect(() => {
-    if (carouselCardContainerRef.current) {
-      setCardWidth(Math.min(carouselCardContainerRef.current.offsetWidth, MAX_CARD_WIDTH))
-    }
-  }, [width])
+const MAX_CARD_WIDTH = 800
```

Comment:
One comment: the difference will be transparent but meaningful from the implementation perspective. In fact, I believe this variable could be renamed (ideas anyone?)

Now, the value of this constant will be used to calculate the offset by which items in the card should be positioned on the x-axis. 

It would seem that having a correct and accurate container width is necessary for carousel to work, but in fact - it is not.

This changes the layout from:
-[item]-[item]-[item]-

to
<-800px->
-[item]----[item]----[item]

this has no impact on the way the carousel works because we're always animating the `x` to 0. The only actual difference is that the animation may feel slightly faster since there's more travel for the card to get into `x = 0` state, but we can always mimick that with slower speed.

Either way, I think this is a great improvement that greatly simplifies the logic here. All possible because we no longer need to manually set `width`, but can use `calc(100%)`.

Reply:
Either way, I think this is a great improvement that greatly simplifies the logic here. However, if the change in speed is not desired/unacceptable, we can bring back previous behavior, so I am marking this as an idea.

Reply:
I honestly felt like it was too slow with the new friction speed but if you could run it by Bryan, Cal, or Yushin on design to verify I'm fine with current behaviour